### DISCUSSION TITLE
[Solved] How to remove the parameters font background box

### DISCUSSION THREAD

**Discussion by 8ar10der** [JanDeDobbeleer/oh-my-posh#3029] [General]
![image](https://user-images.githubusercontent.com/26244249/198907381-05c1398e-9cec-46ea-ad2b-7a00cfde68fb.png)

Hi all,

I try to change lots of the themes and also try to change the terminal. However, the shadow still here. It should be a feature of oh-my-posh. 
Too dark to let me see the font. How can I remove it?

**Comment by JanDeDobbeleer** [comment]
@8ar10der that is a terminal issue. You'll need to create an issue there.

**Comment by 8ar10der** [comment]
@JanDeDobbeleer Hi, I just found an issue maybe about this topic. I am not sure should I open one, so I think discuss it here firstly is better.

I use the same oh-my-posh config, same shell profile and same terminal settings for the PowerShell 7.2.7（store version）and Windows system Powershell 5.1 .

In the store version, the black box background will appear, but in the windows version it will not show.

Store version:
![image](https://user-images.githubusercontent.com/26244249/199316294-2fe52a9f-b9c6-4aba-b7d8-0092437131a0.png)

Windows version:
![image](https://user-images.githubusercontent.com/26244249/199316400-d0381a60-d549-4282-a0fe-8df738e0dd5e.png)

Do you think this is an oh-my-posh support issue or new version powershell issue?

**Reply by JanDeDobbeleer** [reply]
@8ar10der this is PSReadLine who does that, not oh-my-posh. We do not manipulate the commands, only the prompt. Didn't see that at first, but definitely something for PSReadLine.

**Reply by 8ar10der** [reply]
@JanDeDobbeleer Thanks for you mentioned the `PSReadLine`. I just tried to upgrade it to the latest in the pwsh.exe and no issue now.
